src/app/fyle/dashboard/dashboard.page.ts

```
@@ -305,6 +305,36 @@
     ]);
   }
 
+  onExpensesClicked(): void {
+    console.log('Expenses clicked');
```

ESLint

[error] Unexpected console statement.

(no-console)